testing/trino-product-tests-launcher/src/main/resources/docker/trino-product-tests/conf/trino/etc/log.properties

```diff
@@ -0,0 +1,4 @@
+# Reduce levels of some known noisy loggers
+org.eclipse.jetty=ERROR
+org.glassfish.jersey.server.ServerRuntime$Responder=OFF
+io.airlift.concurrent.BoundedExecutor=OFF
```

Comment:
what does it produce, that we want to disable it?

Reply:
When it's shut down it rejects all of the tasks that log a bunch of errors.